### PR TITLE
Fixes color picker segmented control rendering and scrolling issues

### PR DESCRIPTION
**Pull request by antonis**
**Fixes**: https://github.com/WordPress/gutenberg/issues/30878 and https://github.com/wordpress-mobile/gutenberg-mobile/issues/3104

`gutenberg`: https://github.com/WordPress/gutenberg/pull/30994
`WordPress-Android`: https://github.com/wordpress-mobile/WordPress-Android/pull/14492
`WordPress-iOS`: https://github.com/wordpress-mobile/WordPress-iOS/pull/16366

To test:
Follow the steps on the [`gutenberg` pr](https://github.com/WordPress/gutenberg/pull/30994)

PR submission checklist:

- [x] I have considered adding unit tests where possible.
- [x] I have considered if this change warrants user-facing release notes [more info](https://github.com/wordpress-mobile/gutenberg-mobile/blob/develop/Release-notes.md) and have added them to [RELEASE-NOTES.txt](https://github.com/wordpress-mobile/gutenberg-mobile/blob/develop/RELEASE-NOTES.txt) if necessary.

### CODE REVIEW

**Comment by cameronvoell** [comment]
Hey @antonis. 👋🏻 We will cut the 1.52.0 release Friday 30 April. Please let me know if you're aiming to include this PR in 1.52.0. Otherwise perhaps we can add the milestone label for 1.53.0 Thanks! 🙇

File: bundle/ios/GutenbergNativeTranslations.swift [hunk 1]
```diff
@@ -27,39 +27,26 @@ private func dummy() {
     _ = NSLocalizedString("50 / 50", comment: "")
     _ = NSLocalizedString("70 / 30", comment: "")
     _ = NSLocalizedString("9:16", comment: "")
-    _ = NSLocalizedString("A calendar of your site’s posts.", comment: "")
```
Comment (dcalhoun) [inline bundle/ios/GutenbergNativeTranslations.swift]: Do you know if these removals are of consequence? I wonder if they are accurate removals or potentially erroneous changes occurring due to outdated i18n files. Possibly related to https://github.com/wordpress-mobile/gutenberg-mobile/pull/3423? WDYT?

**Review by dcalhoun — APPROVED** [review]
LGTM. Approved via https://github.com/WordPress/gutenberg/pull/30994#pullrequestreview-653922751. I left one question regarding the bundle translation strings, but I think this is ready to merge.

File: bundle/ios/GutenbergNativeTranslations.swift (hunk 1 above)
Comment (geriux) [reply bundle/ios/GutenbergNativeTranslations.swift]: I think we should only merge the Gutenberg reference update and not the rest, I'm guessing they were only present to be able to test the builds right @antonis? If so we can just update this PR to only have the GB commit hash change before merging.

Comment (antonis) [reply bundle/ios/GutenbergNativeTranslations.swift]: Thank you for reviewing this @dcalhoun! I really appreciate the time you put on this 🙇 

> Do you know if these removals are of consequence? I wonder if they are accurate removals or potentially

Nice catch. I'm not sure why the changes were introduced. It should be related with #3423. I've only run `bundle` and not `prebundle` btw 😕 

> I think we should only merge the Gutenberg reference update and not the rest

I would agree with @geriux on this. I would just update the reference. The rest was just needed to be able to produce the App builds for testing.

**Pull request merged**